modules/react/action-bar/index.ts

```diff
@@ -1,5 +1 @@
-import ActionBar from './lib/ActionBar';
-
-export default ActionBar;
```

Comment:
WIll the codemod also update any default imports to named imports? E.g.

```ts
import ActionBar from '@workday/canvas-kit-react/action-bar';
```
becomes this:

```ts
import { ActionBar } from '@workday/canvas-kit-react/action-bar';
```

Reply:
Thanks for catching it! I've updated existing codemod to add a script that changes default import to named or renamed import from `action-bar` package. Now codemod should cover this problem.